src/libsync/owncloudpropagator.cpp

```diff
@@ -939,6 +940,11 @@ void PropagateDirectory::slotSubJobsFinished(SyncFileItem::Status status)
     emit finished(status);
 }
 
+void PropagateDirectory::slotAbortFinished()
+{
+    emit abortFinished();
```

Comment:
Ah no.  i got confused with CompositeJob.

Well, you don't need a slot for this single line, just connect to SIGNAL(abortFinished())